game-of-life-multiplatform/build.gradle

```diff
@@ -6,9 +6,12 @@ allprojects {
         repositories {
             google()
             jcenter()
+            maven { url "https://dl.bintray.com/jetbrains/kotlin-native-dependencies" }
         }
         dependencies {
             classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+            classpath "org.jetbrains.kotlin:kotlin-native-gradle-plugin:+"
+            classpath "org.jetbrains.kotlin:kotlin-native-gradle-plugin:0.6-dev-936"
```

Comment:
We need to use a dev version of the `kotlin-native-gradle-plugin` since the feature has not yet been released officially.